### DISCUSSION TITLE
Question about import command

### DISCUSSION THREAD

**Discussion by ohjays** [Kalanyr/gogrepo#42] [Q&A]
i was trying to import a big collection of mixed folders, but i notice its not importing any of the extras only the installers or is this the intended operation ?
still figuring this out :) thanks

**Comment by Kalanyr** [comment]
A little from column A (it's working as I would expect it too), a little
from column B (I would like it to import extras but that's not really
possible at the moment).

The import works by matching the MD5 hash of each item in the mixed folder
to the MD5 that GOG uses for an item.

Unfortunately GOG doesn't provide MD5s for extras, so there's nothing to
match.

The gogrepoc code does (theoretically) support extras having MD5s if GOG
ever adds them.

**Reply by ohjays** [reply]
right i had a nosey in gog-manifest.dat and saw that, could file size be an option, questionable perhaps but could name and size not give a match for extras.
thanks all the same
